src/dlgMapper.cpp

```diff
@@ -208,15 +208,15 @@ void dlgMapper::slot_toggleShowRoomIDs(int s)
     mp2dMap->update();
 }
 
-void dlgMapper::slot_toggleShowRoomNames(int s)
+void dlgMapper::slot_toggleShowRoomNames(int toggle)
 {
-    mpMap->setRoomNamesShown(s == Qt::Checked);
+    mpMap->setRoomNamesShown(toggle == Qt::Checked);
     mp2dMap->update();
 }
 
-void dlgMapper::slot_toggleStrongHighlight(int v)
+void dlgMapper::slot_toggleStrongHighlight(int toggle)
 {
-    mpHost->mMapStrongHighlight = v == Qt::Checked ? true : false;
+    mpHost->mMapStrongHighlight = toggle == Qt::Checked ? true : false;
```

Comment:
BTW This slot/feature has not been used since Heiko committed a change you made 11 years ago (last day of 2012) 8b0e8bb0f925d58c558cf864faae6b987435bd82 which undid the addition of the feature 2 years before then 3cb1719b0f516eae4fdbd9288e3f8e4e83e9ccbf!
```suggestion
    mpHost->mMapStrongHighlight = (checkState == Qt::Checked) ? true : false;
```